docs/5.x/pack.md

```diff
@@ -834,7 +834,30 @@ installer:
 !!! Note:
 	Currently, only one setup endpoint per application is supported.
 
+## Excluding System Applications
 
+By default, Gravity cluster installs with a number of system applications
+that provide logging, monitoring and application catalog functionality. You
+may want to disable any of these components, for example if you prefer to
+replace them with your own preferred solution. To do that, define the following
```

Comment:
with your own ~~preferred~~ solution.
or
`with a solution of your choice`
There's `prefer` before that and it should be sufficient.

Reply:
Good call, I'll update.